snapm/manager/plugins/stratis.py

```
@@ -242,20 +242,35 @@ def devpath(self):
     @property
     def status(self):
         (_, filesystem) = self._get_dbus_cache()
-        if filesystem.MergeScheduled() is True:
-            return SnapStatus.REVERTING
+        try:
+            if filesystem.MergeScheduled() is True:
+                return SnapStatus.REVERTING
+        except DbusClientMissingPropertyError:
```

Ruff (0.15.9)

[error] Undefined name `DbusClientMissingPropertyError`

(F821)